core/conn/odbc/src/odbc/nsksrvr/Interface/odbcs_srvr_res.cpp

```diff
@@ -642,6 +642,41 @@ odbc_SQLSrvr_EndTransaction_ts_res_(
 
 } // odbc_SQLSrvr_EndTransaction_ts_res_()
 
+CEE_status
+odbc_SQLSrvr_ExtractLob_ts_res_(
+    /* In   */ CEE_tag_def    objtag_
+  , /* In   */ const CEE_handle_def *call_id_
+  , /* In   */ const struct odbc_SQLsrvr_ExtractLob_exc_ *exception_
+  , /* In   */ IDL_long_long  lobDataLen
+  , /* In   */ IDL_char       *lobDataValue
+  )
+{
+    CInterface* pnode = (CInterface *)objtag_;
+
+    CEE_status sts = CEE_SUCCESS;
+
+    IDL_short error;
+    CEERCV_IOMessage_reply_seq_ reply;
+
+    char * buffer = NULL;
+    UInt32 message_length = 0;
+
+    odbc_SQLsrvr_ExtractLob_param_res_(
```

Comment:
If odbc_SQLsrvr_ExtractLob_param_res_ does return error ( CEE_ALLOCFAIL - previous comment ) then store the return status in sts here and check for the same.